package.json

```diff
@@ -114,6 +114,7 @@
     ]
   },
   "resolutions": {
-    "webpack": "^5.59.0"
+    "webpack": "^5.59.0",
+    "@carbon/react": "1.23.0"
```

Comment:
what's happening here? 😅 is this forcing gatsby-theme to use a newer version of react?

Reply:
Yeah, `gatsby-theme-carbon` is resolving to `@carbon/react: 1.11.0`, I told @andreancardona that this may be a temporary workaround until we can update the Gatsby theme to a version that includes `ContainedList`